### DISCUSSION TITLE
How do you remove or control what is shown in the default AppBar

### DISCUSSION THREAD

**Discussion by ruan-rr** [cogentcore/core#1171] [General]
![Screenshot 2024-08-22 162438](https://github.com/user-attachments/assets/825dcfca-f6d5-4ad2-8778-2c4d5131bb5e)

I see you can add buttons to the bar (https://www.cogentcore.org/core/widgets/containers/toolbars), but how do you remove the bar or control the components it contains (`b.GetTopAppBar()` (where b is `core.NewBody()`) returns nil)?

**Comment by kkoreilly** [comment]
This is a duplicate of #1074, which is near the top of my priority list. With the changes I am going to make, you will have much more control over the app bar.
